queries/combine_tables.sql

```diff
@@ -0,0 +1,35 @@
+-- Copyright 2020 Google LLC
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--    http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+-- =============================================================================
+/** Query to combine the cleaned features and the table with nulls removed. */
+
+SELECT 
+  CleanFeatures.*,
+  NullsRemoved.date_received,
```

Comment:
I'm sure I could figure this out if I looked, but why CleanFeatures.* but not NullsRemoved.*?

Reply:
NullsRemoved has the 4 columns - issue, subissue, product and subproduct as well which are not cleaned up since this query just performs the null cleaning functionality.
CleanFeatures has the cleaned up versions of the 4 columns.